yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/commons/DockerPullImage.java

```diff
@@ -161,9 +160,7 @@ public void execInternal(@Nonnull final DockerClient client, @Nonnull final Stri
             }
 
             try {
-                pullImageCmd
-                        .exec(new DockerPullImageListenerLogger(listener))
-                        .awaitSuccess();
+                pullImageCmd.exec(new DockerPullImageListenerLogger(listener));
```

Comment:
without awaitSuccess() it wouldn't wait for full image download

Reply:
awaitSuccess() is deprecated but are you sure it's still required ?